bin/bl-printing-pipemenu

```diff
@@ -60,7 +60,7 @@ else
     # Pipemenu
     menuStart
     if type 'system-config-printer' &> /dev/null; then
-        menuItem 'Configure Printers' 'gksudo system-config-printer'
+        menuItem 'Configure Printers' 'system-config-printer'
```

Comment:
What's John's old commt doing in this PR?

Reply:
No idea, I did specific file pull request (I think).